src/CheckVariables.ts

```diff
@@ -274,6 +274,9 @@ export const checkVariablesOnLoad = (data: Player) => {
         c15RewardUpdate();
         player.loadedDec16Vers = true;
     }
+    if (data.talismanLevels[0] === null) {
```

Comment:
Ah I hadn't see this piece of code, I now see it's because `talismanLevels` and `antUpgrades` are still defined with a `null` at index 0 in the player definition in Synergism.ts that the length check isn't triggered.
Just did a quick test and indeed just removing the nulls was sufficient, I'll remove those 2 unnecessary checks from the PR.